src/NHSD.BuyingCatalogue.Ordering.Domain/Order.cs

```diff
@@ -206,6 +206,11 @@ public int DeleteOrderItem(CatalogueItemId catalogueItemId)
                 || o.CatalogueItem.ParentCatalogueItemId == catalogueItemId);
         }
 
+        public bool HasAdditionalService()
+        {
+            return OrderItems.Any(o => o.CatalogueItem.CatalogueItemType.Equals(CatalogueItemType.AdditionalService));
+        }
```

Comment:
I think this should be `HasAssociatedService` (see above).

Reply:
corrected